src/presence_analyzer/views.py

```diff
@@ -113,14 +113,17 @@ def users_data_view():
     """
     data = get_xml_users()
 
-    return [
+    result = [
```

Comment:
Not necessary just like `data` var above, can be all moved to `return [comprehension]`